doc/index.qdoc

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************************
+**
+** Copyright (C) 2021 Open Mobile Platform LLC
```

Comment:
```suggestion
** Copyright (c) 2021 Open Mobile Platform LLC
```
And same for others, @rainemak what do you think?

Reply:
Thanks, that's a good point. I've fixed this (at least for the files this PR touches).